README.md

```diff
@@ -1,80 +1,6 @@
 # React Native in the Community
 
-A list of community-built React Native contributions. You can subscribe to updates on the highlights of the changes to this repo week-by-week, along with other news from the community in the [react-native-newsletter](https://github.com/brentvatne/react-native-newsletter).
-
-For a searchable list of npm-registered components, please see [react.parts](http://react.parts/native-ios)
-
-## Components
-
-> WARNING: This list is deprecated. The definitive list can be found at [react.parts](http://react.parts/native-ios)
-
-### Audio / Visual
-
-- Camera: https://github.com/lwansbrough/react-native-camera
-- Video: https://github.com/brentvatne/react-native-video
-- Audio Recording: https://github.com/jsierles/react-native-audiorecorder
-
-### Controls / Forms
-
-- Form handling: https://github.com/gcanti/tcomb-form-native
-- UISegmentedControl: http://qiita.com/laiso/items/5c5ef9158a37726d109d
```

Comment:
This isn't in react.parts. Maybe we should keeps a list of components that aren't in react.parts yet